src/python/zquantum/core/utils.py

```diff
@@ -647,3 +647,23 @@ def load_from_specs(specs: Specs):
     if isinstance(specs, str):
         specs = json.loads(specs)
     return create_object(specs)  # type: ignore
+
+
+def get_ordered_list_of_bitstrings(num_qubits: int):
```

Comment:
I'd move it to `measurements.py` instead of keeping it in `utils.py`

Reply:
Why do you think this function is related to measurements? It simply generates orderder binary representation of integers, I put it here as we also have dec2bin and bin2dec here which are similar kind of utils

Reply:
Hmm 🤔 
Everything that has bitstrings looks like measurements to me, but I think you're right :)